Applications/HelloRadium/minimalapp.cpp

```diff
@@ -22,6 +23,8 @@ MinimalApp::MinimalApp( int& argc, char** argv ) :
     m_engine->initialize();
 
     Ra::Gui::KeyMappingManager::createInstance();
+    Ra::Gui::KeyMappingManager::getInstance()->addListener(Ra::Gui::TrackballCamera::registerKeyMapping);
```

Comment:
The clang format style is not respected (should have spaces inside paren)
Same comment many times in this PR

Reply:
I forgot to format the files. This will be done in the next commit.